SDK-Onboarding/readme.md

```diff
@@ -0,0 +1,243 @@
+When writing tests, you need to modify local environment to run tests in live mode, and after doing this, you'll see recoded tests as many file.yml added containing network info for each call. At this point, you can run tests in recoded mode.
+
+Please check your [language-specific section](#language-specific-instructions-1) for how to enable each.  
+
+
+## Final Touches
+At this stage, you need to privde final touches
+
+### Changelog.md
+Update changelog.md file to add entry for the currnt release
+with things like version, date, new features, and breaking changes (if any).
+
+### Readme.md
+This will be the entrypoint for all developers looking for your SDK. You should add things like an overview of what the service does and some samples.
+
+## SDK Release
+At this stage we trigger release pipeline on Azure DevOps to release a package for users (pip package for python, or nuget package for .net).
+
+Check with senior people from SDK team (because you most likely won't have permissions to carry out this step).
+
+## CI/CD Pipeline
+
+### Testing pipeline
+We simply have recorded mode because when submitting a PR or pushing new changes, CI/CD pipeline will only run tests in recorded mode to reduce network traffic. But it also has nightly builds which run tests in live mode. 
+
+### Secrets management
+Every project has a tests.yml file which contains info you need for testing like env variables names used by pipeline.
+These secrets should point to some secret in Azure Keyvault.
+Contact your senior colleague from SDK team for more info.
+
+
+## Resources
+
+### Azure-Core library
+You need to familiarize yourself with the underlying library for all SDKs, which is Azure-Core. You need to be familiar with concepts like Pipelines, LROs, Paging, .. etc.
+
+- Azure-Core .Net: https://docs.microsoft.com/en-us/dotnet/api/overview/azure/core-readme
+
+- Azure-Core Python: https://docs.microsoft.com/en-us/python/api/overview/azure/core-readme?view=azure-python
+
+### Coding Guidelines
+- coding guidelines: https://azure.github.io/azure-sdk/general_introduction.html
+
+### Internal Tutorials
+- Internal Tutorials:           https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/346/Azure-SDK-Development-Training
+- Azure SDK Internal Training:  https://github.com/Azure/azure-sdk-pr/tree/master/training   
+
+### Repositories
+- Azure-api-specs: https://github.com/Azure/azure-rest-api-specs
+- Python SDKs repository: https://github.com/Azure/azure-sdk-for-python/
+- .NET SDKs repository: https://github.com/Azure/azure-sdk-for-net/    
+- Autorest repository: https://www.npmjs.com/package/autorest
+
+
+## Language-specific Instructions
+
+### Python-specific Instructions
+
+#### Prepare development environment
+Create conained environment
+```bash
+python -m venv "path/to/venv"
+```
+
+Install development dependencies
+```bash
+cd "path/to/sdk"
+pip install -r dev_requirements.txt
+```
+
+Install client as package (to run samples and tests)
```

Comment:
you might add that using `-e` makes this an editable install so your install will reflect any changes made in the code